src/components/Dashboard.tsx

```diff
@@ -16,19 +16,24 @@ import {
 import { Octokit } from "octokit";
 import { useDashboardController, useConfiguredRepositoryDetails ,TableItems } from "../classes/DashboardController";
 import {useNavigate} from 'react-router-dom';
+import { Spinner } from '@chakra-ui/react'
 
 
 export default function Dashboard() {
   const dashboardController = useDashboardController();
   const navigate = useNavigate();
   const GitData = useConfiguredRepositoryDetails();
+  const [loading, setLoading] = useState(true)
+
+
 
   useEffect(() => {
     const refreshConfiguredRepositories = () => 
     console.log(dashboardController);
     const octokit = dashboardController.octokit;
     dashboardController.refreshConfiguredRepositories();
-    
+    setTimeout(() => setLoading(false), 100)
```

Comment:
Showing a loading message that automatically stays for a short time and then goes away regardless of when the data is fetched is not ideal. Please either:
1. Change refreshConfiguredRepositories to be async - and use a `.then` in this call to it, where you set `loading` to be `false` when it completes, or
2. Remove the loading message all-together.